src/js/view/controls/components/menu/settings-menu.js

```diff
@@ -304,6 +304,9 @@ class SettingsMenu extends Menu {
     }
 
     open(evt) {
+        const gearButton = this.controlbar.elements.settingsButton.element();
+        gearButton.setAttribute('aria-expanded', true);
```

Comment:
Should this be below the `if (this.visible)`?

Reply:
hmm yeah that does seem like a better spot! Just tested it and it works fine. Will make the suggested change